ledger/src/blockstore_options.rs

```diff
@@ -153,6 +156,38 @@ impl ShredStorageType {
             ShredStorageType::RocksFifo(_) => BLOCKSTORE_DIRECTORY_ROCKS_FIFO,
         }
     }
+
+    /// Returns the ShredStorageType that is used under the specified
+    /// ledger_path.
+    ///
+    /// None will be returned if the ShredStorageType cannot be inferred.
+    pub fn from_ledger_path(
+        ledger_path: &Path,
+        shred_storage_size: u64,
```

Comment:
nit: `shred_storage_size` ==> `fifo_shred_storage_size` (since this arg is only used in fifo case)